md/single-sign-on-with-kerberos.md

```diff
@@ -0,0 +1,331 @@
+
+
+## Troubleshoot
+
+To troubleshoot Kerberos SSO login issues, you need to add a logging handler for the package `net.sourceforge.spnego` and increase the [log level](logging.md) to `ALL` for the packages `org.bonitasoft`, `com.bonitasoft`, and `net.sourceforge.spnego` in order for errors to be displayed in the log files bonita-*.log (by default, they are not).
+
+In order to do that in a Tomcat bundle, you need to edit the file `<BUNDLE_HOME>/server/conf/logging.properties.  
+* Add the lines:  
+```
+net.sourceforge.spnego.handlers = 5bonita.org.apache.juli.AsyncFileHandler
+net.sourceforge.spnego.level = ALL
+```
+* Update the existing lines (to set the level to `ALL`):  
+```
+org.bonitasoft.console.common.server.auth.level = ALL
+org.bonitasoft.engine.authentication.level = ALL
+com.bonitasoft.engine.authentication.level = ALL
+```
+
+In a WildFly bundle, you need to edit the file `<BUNDLE_HOME>/server/standalone/configuration/standalone.xml` in the domain `urn:jboss:domain:logging:3.0` of the *subsystem* tag.
+
+Edit the *logger* tags which *category* matches `org.bonitasoft.console.common.server.auth`, `org.bonitasoft.engine.authentication` and `com.bonitasoft.engine.authentication` packages: change the *level* *name* attribute of each *logger* to `ALL` and add a new logger with the *category* `net.sourceforge.spnego` (also with a *level* *name* set to `ALL`).
+
+## Manage passwords
+
+When your Bonita platform is configured to manage authentication over Kerberos, the users passwords are managed in your AD.  
+However, when you create a user in Bonita Portal, specifying a password is mandatory. This password is ignored when logging in with Kerberos.
+
+## LDAP synchronizer and Kerberos
+
+If you are using an LDAP (or AD) service and the [LDAP synchronizer](ldap-synchronizer.md) to manage your user data,   
+you can continue to do this and manage authentication over Kerberos.  
+The LDAP synchronizer user must be registered in Bonita (no need for an LDAP/AD account). It is recommended though to use the tenant admin account.   
+We recommend that you use LDAP or AD as your master source for information, synchronizing the relevant information with your Bonita platform.
+
+::: info
+**Note :** By default the [LDAP synchronizer](ldap-synchronizer.md) sets the password of the accounts created with the same value as the username. So, even if you allow standard authentication (by setting the property `saml.auth.standard.allowed` in **authenticationManager-config.properties**), users won't be able to log in with the portal login page directly without going through the IdP.   
+:::
+
+## Single sign-on with Kerberos using the REST API
+
+Kerberos is a browser-oriented protocol (based on http automatic redirection, forms, etc...), therefore only resources that require a direct access from a web browser are handled by the Kerberos filter. 
```

Comment:
We should remove everything that's before "only resources that"
Because I don't think Kerberos is a browser-oriented protocol. Only our Kerberos filter is browser oriented